src/test/java/com/jcabi/http/wire/LastModifiedCachingWireTest.java

```diff
@@ -183,4 +183,40 @@ public void cacheUpdateNewerResponse() throws Exception {
             container.stop();
         }
     }
+    /**
+     * LastModifiedCachingWire sends request directly,
```

Comment:
@amihaiemil please see http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-method-names and add `can` to Javadoc